.github/workflows/ci-tests.yml

```diff
@@ -32,40 +32,49 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os:
-          - ubuntu-22.04
-          - macos-12
-          - windows-2022
+        os: ${{ fromJson(github.repository_owner == 'subspace' && github.repository == 'subspace/subspace-cli' && '[["self-hosted", "ubuntu-22.04-x86-64", "Linux"], ["self-hosted", "macos-12-arm64", "macOS"], ["self-hosted", "windows-server-2022-x86-64", "Windows"]]' || '["ubuntu-22.04", "macos-12", "windows-2022"]') }}
```

Comment:
`&& github.repository == 'subspace/subspace-cli'` is unnecessary

Reply:
Using `"self-hosted", "ubuntu-22.04-x86-64", "Linux"` is unnecessary as well, it is very obvious that Ubuntu is Linux, so it should have been `"self-hosted", "ubuntu-22.04-x86-64"`

Reply:
Ok will remove `&& github.repository == 'subspace/subspace-cli'` 

The github runners are funny with labels, and although it's obvious ubuntu is Linux, using "Linux" for conditional statements is cleaner.  i.e `if: runner.os == 'Linux'`